book/src/ci.md

```diff
@@ -11,14 +11,17 @@ jobs:
   cargo-mutants:
     runs-on: ubuntu-latest
     steps:
+      - name: Get number of CPU cores
+        uses: SimenB/github-actions-cpu-cores@v1
+        id: cpu-cores
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
       - name: Install cargo-mutants
         run: cargo install --locked cargo-mutants
       - name: Run mutant tests
-        run: cargo mutants -- --all-features
+        run: cargo mutants --jobs ${{ steps.cpu-cores.outputs.count }} -- --all-features
```

Comment:
On the whole I like the suggestion, but this might make things flaky if running many tasks makes the build VM run out of memory. Could you make this a suggestion in the text rather than putting it into the code people are likely to copy?

Reply:
Looks good, thanks!